src/Uno.WinUIRevert/Program.cs

```diff
@@ -0,0 +1,142 @@
+
+			// Adjust Colors
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UWP\UI\Colors.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UWP\UI\ColorHelper.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\SourceGenerators\Uno.UI.SourceGenerators\XamlGenerator\XamlConstants.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Markup\Reader\XamlConstants.cs"), "Windows.UI", "Microsoft.UI");
+
+			// Custom animation
+			// ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\Animation\Animators\RenderingLoopAnimator.wasm.cs"), "Microsoft", "Windows");
+
+			// Revert partial changes for WinUI 2.4 imported controls
+			//foreach (var file in Directory.EnumerateFiles(Path.Combine(basePath, @"src\Uno.UI\Microsoft\UI\Xaml\Controls"), "*.*", SearchOption.AllDirectories))
+			//{
+			//	ReplaceInFile(file, "namespace Windows.UI.Xaml.Controls", "namespace Windows.UI.Xaml.Controls");
+			//}
+
+			// Restore RadialGradientBrush
+			//ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\RadialGradientBrush.Android.cs"), "namespace Windows.UI.Xaml.Controls", "namespace Windows.UI.Xaml.Controls");
+			//ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\RadialGradientBrush.cs"), "namespace Windows.UI.Xaml.Controls", "namespace Windows.UI.Xaml.Controls");
+			//ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\RadialGradientBrush.iOSmacOS.cs"), "namespace Windows.UI.Xaml.Controls", "namespace Windows.UI.Xaml.Controls");
+			//ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\RadialGradientBrush.wasm.cs"), "namespace Windows.UI.Xaml.Controls", "namespace Windows.UI.Xaml.Controls");
+		}
+
+		static string[] _exclusions = new[] {
+			"Uno.UWPSyncGenerator.Reference.csproj",
+			@"\obj\",
+			@"\bin\",
+			@"\.git",
+			@"\.vs",
+		};
+
+		private static void ReplaceInFolders(string basePath, (string from, string to)[] replacements, string searchPattern = "*.*")
+		{
+			foreach (var file in Directory.EnumerateFiles(basePath, searchPattern, SearchOption.AllDirectories))
+			{
+				if (_exclusions.Any(e => file.Contains(e)))
+				{
+					continue;
+				}
+
+				var updated = false;
+				var content = File.ReadAllText(file);
+
+				for (int i = 0; i < replacements.Length; i++)
+				{
+					if (content.Contains(replacements[i].from))
+					{
+						content = content.Replace(replacements[i].from, replacements[i].to);
+						updated = true;
+					}
+				}
+
+				if (updated)
+				{
+					Console.WriteLine($"Updating [{file}]");
+					File.WriteAllText(file, content);
+				}
+			}
+		}
+
+		private static void DeleteFolder(string path)
+		{
+			if (Directory.Exists(path))
+			{
+				Console.WriteLine($"Deleting {path}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)